main.rb

```diff
@@ -0,0 +1,50 @@
+require './app'
+
+def display_prompts
+  puts 'Welcome to catalog of my things'
+  puts ' '
+  puts 'Choose an option from choices below:'
+
+  puts '1 - List all books'
+  puts '2 - List all music albums'
+  puts '3 - List of games'
+  puts "4 - List all genres (e.g 'Comedy', 'Thriller')"
```

Comment:
[REQUIRED] I noticed that `list all genres` always returns empty. I think this is because there is no way to create the genre. Kindly include it. You can ask to include genre while creating books or music or both.